pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -99,6 +94,13 @@ title: <%= I18n.t('dashboard_landing_title').inspect %>
     return {
       templateUrl: 'teacher-dashboard/nav',
       controller: function($scope) {
+        // Show projects tab in the section nav if any of the following are true:
+        // 1. CDO.section_projects is set
+        // 2. DCDO.section_projects is set
+        // 3. sectionProjects url param is present
+        // 4. the projects tab is specified in the url
+        // The projects tab is accessible by direct url regardless of settings 1-3 above.
+        $scope.showSectionProjectsTab = #{show_section_projects} || $scope.tab === 'projects';
```

Comment:
I'm not sure how interspersed these sorts of things are in our angular code, but if we're potentially going to refactor this one day, I wonder if it makes sense to try and keep our JS as pure as we can, and have all embedded ruby in one place, i.e. do something like

```
// at the beginning of the file
var showSectionProjects = #{show_section_projects};
...
// now here
$scope.showSectionProjectsTab = showSectionProjects || $scope.tab === 'projects';
``